widgets/matplotlib/simulation/recoil_atom_distribution.py

```diff
@@ -1248,10 +1254,14 @@ def update_recoil_element_info_labels(self):
         self.parent.nameLabel.setText(
             f"Name: {self.current_recoil_element.name}")
 
-        density = self.current_recoil_element.reference_density
+        if self.current_element_simulation.request\
+                .manual_reference_density_checked:
+            density = self.current_recoil_element.reference_density
+        else:
+            density = self.current_recoil_element.reference_density_element_default
```

Comment:
Crashes if a new sub-element is added in *Recoil atom distribution*:
```
Traceback (most recent call last):
  File "<potku>\widgets\matplotlib\simulation\recoil_atom_distribution.py", line 1034, in choose_element
    self.update_recoil_element_info_labels()
  File "<potku>\widgets\matplotlib\simulation\recoil_atom_distribution.py", line 1261, in update_recoil_element_info_labels
    density = self.current_recoil_element.reference_density_element_default
AttributeError: 'RecoilElement' object has no attribute 'reference_density_element_default'
```

Reply:
A good catch.